playground/mocks/idp/idx/sso-extension.js

```diff
@@ -0,0 +1,5 @@
+const templateHelper = require('../../../config/templateHelper');
+
+module.exports = templateHelper({
+  path: '/idp/idx/authenticators/sso_extension/transactions/123/verify',
```

Comment:
we shall be able to write path like `.../sso_extension/transactions/:transactionId/verify` instead of hard code id.